src/ansys/mechanical/core/embedding/logger/linux_api.py

```diff
@@ -0,0 +1,134 @@
+"""Internal mechanical logging linux API.
```

Comment:
```suggestion
"""Internal Mechanical logging Linux API.
```
There shouldn't be more than two adjectives before a noun. This has four! Perhaps change to "Linux API forinternal Mechanical logging"?